frontend/src/components/organisms/MatchesList/MatchListItem.tsx

```diff
@@ -0,0 +1,20 @@
+export type MatchListItem = {
+    onClickTeam: () => void,
+    matchNumber: number | undefined,
```

Comment:
Nie podoba mi sie to wszystko undefined? Dlaczego tak jest? Moze byc mecz bez numeru / statusu? Czy team bez graczy?

Reply:
Nie ma niektórych danych wysyłanych na backendzie :)